src/bindings/scala/project/plugins.sbt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2021 Concurrent Technologies Corporation
```

Comment:
Copyright 2021-2022 Concurrent Technologies Corporation

Reply:
How was the decision made to track this as a list?

https://www.copyrightlaws.com/copyright-symbol-notice-year/